react/reducer/index.ts

```diff
@@ -95,10 +96,18 @@ function reducer(
 
       return {
         ...state,
+        loadingItem: false,
         selectedItem: args.item,
       }
     }
 
+    case 'SET_LOADING_ITEM': {
+      return {
+        ...state,
+        loadingItem: !!action.args.loadingItem,
```

Comment:
Just to make it clearer :)

```suggestion
        loadingItem: Boolean(action.args.loadingItem),
```